src/components/ProfileSettings.js

```diff
@@ -444,7 +446,8 @@ class ProfileSettings extends Component {
     }
     const mapDispatchToProps = dispatch => {
       return {
-        updateProfile : (payload) => dispatch(updateProfile(payload))
+        updateProfile : (payload) => dispatch(updateProfile(payload)),
+        setLocale : (locale) => dispatch(setLocale(locale)),
       }
     }
     export default connect(mapStateToProps, mapDispatchToProps) (ProfileSettings);
```

Comment:
New line

Reply:
@AJAkimana Thanks for that insight, I added the new line like so